lib/services/role_test.go

```diff
@@ -3696,6 +3643,14 @@ func makeTestServer(labels map[string]string) types.Server {
 	return s
 }
 
+func makeTestWindowsDesktop(labels map[string]string) types.WindowsDesktop {
```

Comment:
nit: a `must` prefix is often used to indicate that the func panics on error

```suggestion
func mustMakeTestWindowsDesktop(labels map[string]string) types.WindowsDesktop {
```